django/contrib/admin/static/admin/css/forms.css

```diff
@@ -106,12 +108,6 @@ form .aligned ul {
     padding-left: 10px;
 }
 
-form .aligned ul.radiolist {
-    display: inline-block;
```

Comment:
Matthias has already caught the absence of this, which is causing the wrapping, because the adjacent `label` is `float: left` - if you inspect the `.radiolist` element without this line, you'll see that it spans the whole width, including the `label` itself.